chapter_linear-networks/softmax-regression-scratch.md

```diff
@@ -245,6 +271,22 @@ def accuracy(y_hat, y):  #@save
     return float(d2l.reduce_sum(d2l.astype(cmp, y.dtype)))
 ```
 
+```{.python .input}
```

Comment:
it would be better if we can reuse this function.

Reply:
paddle.Tensor does not support 0-dimension operation, so it is easy to add a layer of dimension when using it. Here, the size must be judged once. If the size is wrong, it is likely to cause calculation errors.

Reply:
paddle.Tensor does not support 0-dimension operation, so it is easy to add a layer of dimension when using it. Here, the size must be judged once. If the size is wrong, it is likely to cause calculation errors.